### PR TITLE
Monthly Checklist

### PR DESCRIPTION
**Pull request by rdohms**
In the last commit here, we create the template for monthly meetings, please review and suggest other tasks.

### CODE REVIEW

File: config/checklists/monthly-meeting.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+lists: default
+cards:
+  - name: Update Meetup
+    desc: Adjust hoster and speaker information on meetup.com
+    due: -4 weeks
+  - name: Announce Meetup
+    desc: Click the announce button
+    due: -4 weeks
+  - name: Prepare Slides
+    desc: Click the announce button
```
Comment (lcobucci) [inline config/checklists/monthly-meeting.yaml]: This should be `Create PR with slides for next meeting` or something

File: config/checklists/test-template.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+lists: default
+cards:
+  - name: Update Meetup
+    desc: Adjust hoster and speaker information on meetup.com
+    due: -4 weeks
+  - name: Announce Meetup
+    desc: Click the announce button
+    due: -4 weeks
+  - name: Prepare Slides
+    desc: Click the announce button
```
Comment (lcobucci) [inline config/checklists/test-template.yaml]: This should be `Create PR with slides for next meeting` or something

File: src/Command/CreateBoards.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+        $this->announceNewBoardOnSlack($result);
+
+        if ($result->isSuccess()) {
+            $output->writeln(sprintf('<info>%s</info>', $result->getMessage()));
+            $output->writeln('');
+            $output->writeln(sprintf('<comment>Board Url:</comment> %s', $result->getContext()['board_url'] ?? '-n/a-'));
+            $output->writeln(sprintf('<comment>From Template:</comment> %s', $result->getContext()['template'] ?? '-n/a-'));
+            $output->writeln(sprintf('<comment>For Meetup on:</comment> %s', $result->getContext()['date']->format(DATE_RFC3339) ?? '-n/a-'));
+        } else {
```
Comment (lcobucci) [inline src/Command/CreateBoards.php]: OMG @rdohms is using `else`!!!!! This is a major event.
I think he just did it to see if people would notice :rofl:

File: src/Trello/BoardManager.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+use const DATE_RFC3339;
+
+final class BoardManager
+{
+    /**
+     * @var Client
+     */
+    private $trello;
+
+    private $organizationId;
```
Comment (lcobucci) [inline src/Trello/BoardManager.php]: Is `$organizationId` an object? A string? An integer? A bunch of bananas? This is a mistery!!

**Review by lcobucci — APPROVED** [review]
Looks good to me. The checklist is quite sane and has everything I can think of now.

There're a few nitpicks (surprise!) but not to worry we can solve them with tooling.

File: config/checklists/test-template.yaml (hunk 2 above)
Comment (lcobucci) [reply config/checklists/test-template.yaml]: @rdohms I also saw now that this hasn't been updated with the latest state of `monthly-meeting.yaml`. Did you miss it or is this file not being used anymore?

File: src/Slack/Announcer.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace AmsterdamPHP\TrelloChecklister\Slack;
+
+use AmsterdamPHP\TrelloChecklister\ExecutionResult;
+
+final class Announcer
+{
+    public function announceOperationResult(ExecutionResult $result): void
+    {
+
```
Comment (WyriHaximus) [inline src/Slack/Announcer.php]: Is it screaming into the void?

File: src/Trello/BoardManager.php (hunk 4 above)
Comment (WyriHaximus) [reply src/Trello/BoardManager.php]: PHP 7.4 pretty please

**Review by WyriHaximus — CHANGES_REQUESTED** [review]
Need help crafting the unit tests?

**Comment by rdohms** [comment]
@WyriHaximus i think we can skip tests for now.. i rather get functionality ready.

**Review by WyriHaximus — APPROVED** [review]
@rdohms :+1:

**Pull request merged**